Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-notifications.md

```diff
@@ -66,50 +66,67 @@ spec:
     + port defaults to: 5672/5671 for AMQP/S accordingly
     + user/password defaults to: guest/guest
     + user/password may only be provided if HTTPS is used with the RGW. If not, topic creation request will be rejected
     + vhost defaults to: “/”
-12. `disableVerifySSL` (optional) indicates whether the RGW is going to verify the SSL certificate of the AMQP server in case AMQPS is used ("false" by default)
+12. `disableVerifySSL` (optional) indicates whether the RGW is going to verify the SSL certificate of the AMQP server in case AMQPS is used (`false` by default)
 13. `ackLevel` (optional) indicates what kind of ack the RGW is waiting for after sending the notifications:
-    + “none”: message is considered “delivered” if sent to broker
-    + “broker”: message is considered “delivered” if acked by broker (default)
-    + “routable”: message is considered “delivered” if broker can route to a consumer
+    + `none`: message is considered delivered if sent to broker
+    + `broker`: message is considered delivered”if acked by broker (default)
+    + `routable`: message is considered delivered”if broker can route to a consumer
 14. `exchange` in the AMQP broker that would route the notifications. Different topics pointing to the same endpoint must use the same exchange
 15. `kafka` (optional) hold the spec for a Kafka endpoint. The format of the URI would be: `kafka://[<user>:<password>@]<fqdn>[:<port]`
     + port defaults to: 9092
     + user/password may only be provided if HTTPS is used with the RGW. If not, topic creation request will be rejected
     + user/password may only be provided together with `useSSL`, if not, the connection to the broker would fail
-16. `disableVerifySSL` (optional) indicates whether the RGW is going to verify the SSL certificate of the Kafka server in case `useSSL` flag is used ("false" by default)
+16. `disableVerifySSL` (optional) indicates whether the RGW is going to verify the SSL certificate of the Kafka server in case `useSSL` flag is used (`false` by default)
 17. `ackLevel` (optional) indicates what kind of ack the RGW is waiting for after sending the notifications:
-    + “none”: message is considered “delivered” if sent to broker
-    + “broker”: message is considered “delivered” if acked by broker (default)
-18. `useSSL` (optional) indicates that secure connection will be used for connecting with the broker (“false” by default)
+    + `none`: message is considered“delivered”if sent to broker
+    + `broker`: message is considered“delivered”if acked by broker (default)
+18. `useSSL` (optional) indicates that secure connection will be used for connecting with the broker (`false` by default)
 19. `mechanism` (optional) select which authentication mechanism to use, one of:
-    + “PLAIN” (default)
-    + “SCRAM-SHA-512”
-    + “SCRAM-SHA-256”
-    + “GSSAPI”
-    + “OAUTHBEARER”
-
-!!! note
```

Comment:
This note was removed as it seems to be out of date as the behavior is selectable via `ackLevel`.